### PR TITLE
Invoicer

### LINKED ISSUE #46: Add in support for invoicer
**Requirements**

- [x] Config files
- [x] proposed supporting filesystem infrastructure
- [x] docker-compose file

### PR DESCRIPTION
**Pull request by AnotherDroog**
This adds a neutrino-compatible `invoicer` container.

In order to reduce bandwidth we are also setting `assumechanvalid=1` in `lnd.conf`

TODO: Fetch `invoicer-ui/index.html` and place in `public_html` automatically, e.g. using `check_to_fetch()` during startup

This closes #46

### CODE REVIEW

**Comment by nolim1t** [comment]
lgtm

**Comment by nolim1t** [comment]
I also recommend including [these changes](https://github.com/lncm/pi-factory/pull/234) into the composefile too as ```routing.assumechanvalid=1``` does not work without the ```experimental``` flag built

File: install.sh [hunk 1]
```diff
@@ -14,20 +17,56 @@ alpine_install() {
...
 }
 
+debian_install() {
+    if ! command -v apt-get >/dev/null 2>&1; then
+        echo "Error: apt-get not available"
+        return 1
+    fi
+
+    if ! command -v docker >/dev/null 2>&1; then
+        # only install docker if missing
```
Comment (meeDamian) [inline install.sh]: NIT: move comment before `if`

File: install.sh [hunk 2]
```diff
@@ -57,33 +96,65 @@ chroot_install() {
...
+    /alpine-v310/enter-chroot ./install.sh
+}
+
+check_os() {
+    if [ -f "/etc/alpine-release" ]; then
+        alpine_install
+        return 0
+    fi
+
+    if [ -x "$(command -v apt-get)" ]; then
```
Comment (meeDamian) [inline install.sh]: https://github.com/lncm/noma/pull/48/files#diff-3fbb47e318cd8802bd325e7da9aaabe8R31 checks for the same thing, but in a different way.  Perhaps it's better to decide on one way?

File: lnd/neutrino/lnd.conf [hunk 3]
```diff
@@ -68,6 +68,10 @@ tlsextradomain=lnd
 tlsextraip=127.0.0.1
 tlsextraip=192.168.83.33
 
+[Routing]
+; Save bandwidth by not downloading legacy channel filters
```
Comment (meeDamian) [inline lnd/neutrino/lnd.conf]: "legacy"?

**Comment by meeDamian** [comment]
Note that after https://github.com/lncm/invoicer/pull/43 is merged-in invoicer version gets bumped to `v0.6.0`.

**Pull request merged**